homeassistant/components/lock/__init__.py

```diff
@@ -112,6 +112,7 @@ class LockEntity(Entity):
     _attr_is_unlocking: bool | None = None
     _attr_is_jammed: bool | None = None
     _attr_state: None = None
+    _attr_supported_features: LockEntityFeature | int = 0
```

Comment:
Small note for reviewers: this changes default from `None` to `0`, to align with other platforms.